blackbox/bwc/create_bwc_index.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+import os
+import subprocess
+from crate.testing.layer import CrateLayer
+from crate.client import connect
+
+
+CRATE_HTTP_PORT = '42222'
+CRATE_TRANSPORT_PORT = '43333'
+
+
+CREATE_INDEX_SQL = """
+    CREATE TABLE legacy_geo_point (
+        id int primary key,
+        p geo_point
+    ) CLUSTERED INTO 1 SHARDS WITH (
+        number_of_replicas=0,
+        "translog.flush_threshold_ops"=0,
+        "gateway.local.sync"=0
+    );
+    INSERT INTO legacy_geo_point (id, p) VALUES (1, 'POINT (10 10)');
+    REFRESH TABLE legacy_geo_point;
+"""
+
+
+def compress_index(version, data_dir, output_dir):
+    compress(data_dir, output_dir, 'bwc-index-%s.zip' % version, 'data')
+
+
+def compress(data_dir, output_dir, zipfile, directory):
+    abs_output_dir = os.path.abspath(output_dir)
+    zipfile = os.path.join(abs_output_dir, zipfile)
+    if os.path.exists(zipfile):
+        os.remove(zipfile)
+    olddir = os.getcwd()
+    os.chdir(os.path.dirname(data_dir))
+    subprocess.check_call('zip -r %s %s' % (zipfile, directory), shell=True)
```

Comment:
Can we use the `cwd` param of Popen/check_call instead of doing the chdir stuff manually?
And please try to avoid `shell=True`